### PR TITLE
Implement `get_hist_total`

### LINKED ISSUE #15: Add a get_hist_total method wrapper
- [ ] Create a method/function to wrap the method `DataBase.get_hist_total()`.
- [ ] The outputs should be stored in a dictionary with three three values date, state, and counts; each of them a numpy array.
- [ ] Create a test making sure this runs, i.e., run a simulation, call the function, and check the lengths of the elements (all three should match).
- [ ] Add an example in the `README.qmd` that uses the functions and plots the counts of each time series (suscept, infec, rec) using matplotlib.

### PR DESCRIPTION
**Pull request by IsaccBarker**
Fixes #15.

---

Added database tests by way of running a SEIR model and checking the lengths of the returned Numpy arrays `dates`, `states`, and `counts` to make sure they are of the correct length.

---

We expose geting the database from models with the `getDb` function.
    
We expose getting historical totals from databases with the `getHistTotal` function, which returns a dictionary of dates, states, and data (counts).
    
We expose getting the name of a model with the `getName` function.

---

Added a graph to the example in the `qmd` file. This pulls a generated graph from the `README_files` directory.

### CODE REVIEW

**Comment by IsaccBarker** [comment]
For reference, a generated graph. I made the y-axis logarithmic because I felt it visualized things better, but if this is unacceptable let me know.
image

**Comment by gvegayon** [comment]
Awesome job! Notice there's a lot of files that should not be included, namely, all the HTML stuff it is only used for pre-view the HTML version of the README.md file.

**Comment by gvegayon** [comment]
> For reference, a generated graph. I made the y-axis logarithmic because I felt it visualized things better, but if this is unacceptable let me know. Also, the y-axis is labeled `count`. I assume there's a better label, but I'm unsure as to what it might be. image

Looks great! I would keep the "regular scale" and perhaps remove the susceptible cases only.

**Comment by IsaccBarker** [comment]
> Awesome job! Notice there's a lot of files that should not be included, namely, all the HTML stuff it is only used for pre-view the HTML version of the README.md file.

Sorry about that; I generated an HTML report and must have forgotten to remove that data. Will remove.

**Comment by IsaccBarker** [comment]
Graph sample
image

**Comment by gvegayon** [comment]
The Codacity statement seems to be a common issue: https://github.com/fossasia/query-server/issues/332.

File: README.md [hunk 1]
```diff
@@ -4,11 +4,10 @@
 [![Pip
 Build](https://github.com/UofUEpiBio/epiworldpy/actions/workflows/pip.yaml/badge.svg)](https://github.com/UofUEpiBio/epiworldpy/actions/workflows/pip.yaml)
 
-This is a python wrapper of the [`epiworld c++`
-library](https://github.com/UofUEpiBio/epiworld), an ABM simulation
-engine. This is possible using the
-[`pybind11`](https://pybind11.readthedocs.io/en/stable/) library (which
-rocks!).
+This is a python wrapper of the \[`epiworld c++`
+library\]\[epiworld-git\], an ABM simulation engine. This is possible
```
Comment (gvegayon) [inline README.md:8]: This line is not rendering properly. See the README:

image

Comment (gvegayon) [reply README.md:8]: Fix that and we are ready to merge!

**Pull request merged**